arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -525,6 +528,28 @@
           "description": "An array of administrator user identities."
         }
       }
+    },
+    "BackupConfiguration": {
+      "description": "An object that represents backup configurations",
+      "type": "object",
+      "properties": {
+        "storageAccount": {
+          "type": "string",
+          "description": "The name of storage account for backup configuration. It is in the format of 'Resource Group Name/Storage Account Name'"
```

Comment:
What does "in the near future" mean in this context? Can SDKs be generated (and shipped) based on the current behavior, and the API version will be updated for the planned changed (correct) behavior? In general, we want to avoid merging swagger specs that we know will change (unless the change includes an API version bump) into master since it makes the SDK release pipeline very difficult to manage.